restler/unit_tests/authentication_test_files/unit_test_server_auth_module.py

```diff
@@ -0,0 +1,4 @@
+def acquire_token(log):
```

Comment:
add copyright

# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.

Reply:
added